software/rallly.yml

```
@@ -4,10 +4,13 @@ description: Create polls to vote on dates and times (alternative to Doodle).
 licenses:
   - AGPL-3.0
 platforms:
-  - Nodejs
+  - Node.js
```

```suggestion
  - Nodejs
```

The previous syntax was correct https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/platforms/nodejs.yml